src/test/groovy/ee/tuleva/onboarding/mandate/MandateFixture.groovy

```diff
@@ -228,6 +228,7 @@ class MandateFixture {
     mandate.setId(123)
     mandate.setCreatedDate(Instant.parse("2021-03-10T12:00:00Z"))
     mandate.setMandate("file".getBytes())
+    mandate.setMetadata(Map.of())
```

Comment:
In idiosyncratic groovy you can just do `[:]` for an empty map instead of the more Java-like `Map.of()`